janusgraph-core/src/main/java/org/janusgraph/graphdb/tinkerpop/JanusGraphDefaultSchemaMaker.java

```diff
@@ -34,6 +40,39 @@ public Cardinality defaultPropertyCardinality(String key) {
         return Cardinality.SINGLE;
     }
 
+    @Override
+    public PropertyKey makePropertyKey(PropertyKeyMaker factory, Object value) {
+        String name = factory.getName();
+        Class klass = null;
+        if (value instanceof String) {
+            klass = String.class;
+        } else if (value instanceof Character) {
+            klass = Character.class;
+        } else if (value instanceof Boolean) {
+            klass = Boolean.class;
+        } else if (value instanceof Byte) {
+            klass = Byte.class;
+        } else if (value instanceof Short) {
+            klass = Short.class;
+        } else if (value instanceof Integer) {
+            klass = Integer.class;
+        } else if (value instanceof Long) {
+            klass = Long.class;
+        } else if (value instanceof Float) {
+            klass = Float.class;
+        } else if (value instanceof Double) {
+            klass = Double.class;
```

Comment:
The supported data types in [Table 5.1](http://docs.janusgraph.org/0.1.0-SNAPSHOT/schema.html#_defining_property_keys) lists also `Decimal` and `Precision`. Here's the set from [`StandardSerializer.java`](https://github.com/JanusGraph/janusgraph/blob/master/janusgraph-core/src/main/java/org/janusgraph/graphdb/database/serialize/StandardSerializer.java#L76-L92)

It doesn't look like those 2 are valid anymore, so please remove those from the doc.